### PR TITLE
react1-week2 HW

### PR DESCRIPTION
**Pull request by slastushevskyi**

### CODE REVIEW

File: react/react1/week2/vite-project/src/LifeCycleComponent.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React, { useState, useEffect } from "react";
+
+function countSeconds() {
+  const [counter, setCounter] = useState(0);
+
+  useEffect(() => {
```
Comment (LucasIversen) [inline react/react1/week2/vite-project/src/LifeCycleComponent.jsx:6]: nice 👍

File: react/react1/week2/vite-project/src/TodoListComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      description: "Brush teeth",
+      crossOut: "false",
+    },
+    {
+      id: 2,
+      description: "Eat breakfast",
+      crossOut: "false",
+    },
+  ]);
+  const [todoId, setTodoId] = useState(3);
```
Comment (LucasIversen) [inline react/react1/week2/vite-project/src/TodoListComponent.jsx:22]: Maybe we could have just used `todo.slice(-1).id + 1`, to get the id of the last todo, and then increment it by one

File: react/react1/week2/vite-project/src/TodoListComponent.jsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    const newTodo = { id: todoId, description: fetchedTodo, crossOut: "false" };
+    setTodo([...todo, newTodo]);
+  }
+  return (
+    <>
+      <button onClick={AddTodoItem}>Create Random Todo</button>
+      {todo.length > 0 ? (
+        <ul>
+          {todo.map((item) => {
+            return (
```
Comment (LucasIversen) [inline react/react1/week2/vite-project/src/TodoListComponent.jsx]: Let's put this in a separate component

**Review by LucasIversen — CHANGES_REQUESTED** [review]
Hi Nikita. Good job 👍 Just a few requested changes from here to make it perfect

File: react/react1/week2/vite-project/src/TodoListComponent.jsx (hunk 2 above)
Comment (slastushevskyi) [reply react/react1/week2/vite-project/src/TodoListComponent.jsx:22]: Maybe, but I was thinking another way.
I'll do it next time!

File: react/react1/week2/vite-project/src/TodoListComponent.jsx (hunk 3 above)
Comment (slastushevskyi) [reply react/react1/week2/vite-project/src/TodoListComponent.jsx]: Did that. Thank you for review🙏

**Pull request merged**